index.bs

```diff
@@ -866,7 +866,7 @@ When the {{MLContext/[[contextType]]}} is set to [=context type/default=] with t
     1. [=map/For each=] |name| → |outputValue| of |outputs|:
         1. Issue a compute request to |graph|.{{MLGraph/[[implementation]]}} given |name| and |inputResources| and wait for completion.
             1. If that returns an error, then return an "{{OperationError}}" {{DOMException}}.
-            1. Otherwise, store the result in |outputTensor|.
+            1. Otherwise, let |outputTensor| be the result.
```

Comment:
🤔 We have a mix of various ways of *setting* variables, from an explicit "set" and "store" to passive voice "let".

- Set |descriptor|.{{MLOperandDescriptor/dimensions}} to |outputShape|.
- Let |desc| be a new {{MLOperandDescriptor}} initialized to |input|.{{MLOperand/[[descriptor]]}}.
- Otherwise, store the result in |outputTensor|.
- Otherwise, let |outputTensor| be the result.

I suppose a "set" is used when something already exists, and a "let" is used when something new is being *declared* *and* set?

Reply:
Correct. Per [Infra](https://infra.spec.whatwg.org/#variables)

> A variable is declared with "let" and changed with "set".

I have a lint tool able to scrub for variables, and it ensures we're using either "let" or "for each" at least once per variable in each algorithm, to avoid "uninitialized" usage. With these changes, the spec is clean. I'll put up a "tools" PR soon.

Reply:
✅